source/sourcing_and_processing_data/harvesters/dbnomics.rst

```diff
@@ -0,0 +1,37 @@
+DB.nomics harvester
+=========================
+
+Retrieve one or more datasets from `DB.nomics <https://db.nomics.world/>`_.
+
+A dataset is an aggregation of time series. Using the provider name and the dataset ID, it is possible to either fetch each of these time series individually as one OpenDataSoft dataset or keep them aggregated in a single OpenDataSoft dataset.
```

Comment:
I would say `"It is possible to either fetch each of these time series as individual OpenDataSoft datasets or aggregate them as a single OpenDataSoft dataset. The aggregated dataset will have an additional "category" column with the origin of each row."` What do you think?